trio/_threads.py

```diff
@@ -425,7 +427,16 @@ def from_thread_run_sync(fn, *args, trio_token=None):
     def callback(q, fn, args):
         @disable_ki_protection
         def unprotected_fn():
-            return fn(*args)
+            call = fn(*args)
```

Comment:
This is the value returned by the function, so `call` is not a very clear name for it. `result` or `res` or `ret` perhaps?